components/bookshelf.vue

```
@@ -0,0 +1,37 @@
+
```

ESLint

[error] Component name "bookshelf" should always be multi-word.

(vue/multi-word-component-names)